public/embed-application.js

```diff
@@ -128,7 +135,21 @@ function cleanUpVisierSession() {
  */
 function attachSessionEventHandlers(embeddingApp) {
     embeddingApp.on("session", function(msg) {
+        console.log("SESSION: ", msg);
         switch (msg?.code?.toUpperCase()) {
+            case "EJECT_SESSION":
+                /**
+                 * This message is emitted if another tab is opened which embeds content for a different user or tenant.
```

Comment:
This message is emitted if the user opens another tab that embeds content for a different user or tenant.